src/app/client/src/app/app.component.html

```diff
@@ -3,8 +3,8 @@
     [buttonLabel]="resourceService.frmelmnts.btn.submit" (submit)="updateFrameWork($event)"></app-popup>
   <app-tnc-popup (close)="onAcceptTnc()" #termsAndCondPopUp *ngIf="showTermsAndCondPopUp && !isOffline">
   </app-tnc-popup>
-  <app-validate-teacher-identifier-popup *ngIf="flag"> </app-validate-teacher-identifier-popup>
-  <app-user-location *ngIf="!isLocationConfirmed" [isCustodianOrgUser]="isCustodianOrgUser" [deviceProfile]="deviceProfile" [userProfile]="usersProfile">
+  <app-validate-teacher-identifier-popup *ngIf="showUserVerificationPopup" [userFeedData]="userFeed"> </app-validate-teacher-identifier-popup>
```

Comment:
Do we need two variable to check can't we use the userFeed object for the if condition also?